docs/source/developer_guide/plugins/basics.rst

```diff
@@ -4,72 +4,28 @@ Basics
-
-* **Ease of use**: plugins can be found in an online curated list of plugins and installed with one simple command. This process is familiar to every regular python user.
-
-* **Decouple development and update cycles of AiiDA and plugins**: since plugins are separate python packages, they can be developed in a separate code repository and updated when the developer sees fit without a need to update AiiDA. Similarly, if AiiDA is updated, plugins may not need to release a new version.
-
-* **Promote modular design in AiiDA development**: separating plugins into their own python packages ensures that plugins can not (easily) access parts of the AiiDA code which are not part of the public API, enabling AiiDA development to stay agile. The same applies to plugins relying on other plugins.
-
-* **Low overhead for developers**: plugin developers can write their extensions the same way they would write any python code meant for distribution.
-
-* **Automatic AiiDA setup and testing of plugins**: installation of complete python environments consisting of many packages can be automated, provided all packages use ``setuptools`` as a distribution tool. This enables use of AiiDA in a service-based way using, e.g., docker images. At the same it becomes possible to create automated tests for any combination of plugins, as long as the plugins provide test entry points.
-
-
-Design guidelines
-------------------
-
-* **Start simple.**: make use of existing classes like :py:class:`~aiida.orm.nodes.process.calculation.calcjob.CalcJobNode`, :py:class:`~aiida.orm.nodes.data.dict.Dict`, :py:class:`~aiida.orm.nodes.data.singlefile.SinglefileData`, ... Write only what is necessary to pass information from and to AiiDA.
-
-* **Don't break data provenance.**: store *at least* what is needed for full reproducibility.
-
-* **Parse what you want to query for.**: make a list of which information to:
-
-  #. parse into the database for querying (:py:class:`~aiida.orm.nodes.data.dict.Dict`, ...)
-  #. store in files for safe-keeping (:py:class:`~aiida.orm.nodes.data.singlefile.SinglefileData`, ...)
-  #. leave on the remote computer (:py:class:`~aiida.orm.nodes.data.remote.RemoteData`, ...)
-
-* **Expose the full functionality.**: standardization is good but don't artificially limit the power of a code you are wrapping - or your users will get frustrated. If the code can do it, there should be *some* way to do it with your plugin.
-
-
 What a plugin can do
 --------------------
 
-* Add new classes to AiiDA's unified interface, including:
-
-  - calculations
-  - parsers
-  - data types
-  - schedulers
-  - transports
-  - db importers
-  - db exporters
-  - subcommands to some ``verdi`` commands
-
+* Add a new class to AiiDA's :ref:`entry point groups <plugins.entry_point_list>`, including:: calculations, parsers, workflows, data types, verdi commands, schedulers, transports and importers/exporters from external databases.
```

Comment:
Maybe add
> See the output of `verdi plugin list` for the available categories and to inspect which entry points are currently installed for each category

Reply:
thanks for the suggestion, I'll add it instead to the linked section on entry points